README.rst

```diff
@@ -1,18 +1,44 @@
+=============
 aldryn-disqus
 =============
 
-Integrate `Disqus <http://disqus.com>`_ into your `django CMS <http://django-cms.org>`_ and `Aldryn projects 
-<http://aldryn.com>`_, and allow users to comment on and discuss content on your site.
+Integrate `Disqus <http://disqus.com>`_ into your
+`django CMS <http://django-cms.org>`_ and `Aldryn projects <http://aldryn.com>`_,
+and allow users to comment on and discuss content on your site.
 
-Disqus is one of the most popular commenting systems available. It’s especially suited to news and weblog content,
-but can be applied anywhere that you’d like to provide discussion functionality.
+Disqus is one of the most popular commenting systems available. It’s especially
+suited to news and weblog content, but can be applied anywhere that you’d like
+to provide discussion functionality.
 
-With the Aldryn Disqus Addon you can integrate it into your projects and start building an online community in just a
-few simple steps.
+With the Aldryn Disqus Addon you can integrate it into your projects and start
+building an online community in just a few simple steps.
 
 `See it in action on the django CMS weblog <http://www.django-cms.org/en/blog/>`_.
 
+Installation
+============
+
+Aldryn Installation
+-------------------
+
+Choose a site you want to install the add-on to from the dashboard. Then go to
+``Apps -> Install app`` and click ``Install`` next to ``Aldryn Disqus`` app.
```

Comment:
it's probably worth mentioning that the equivalent of DISQUS_SHORTNAME has to be set in configuration for the aldryn addon